Makefile

```diff
@@ -1,46 +1,56 @@
 EXTENSION-LIBRARY-FOLDER-NAME = places
+TEST-APP-FOLDER-NAME = testapp
 
-checkstyle:
-		(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) checkstyle)
+init:
+	git config core.hooksPath .githooks
 
 clean:
-	  (./code/gradlew -p code clean)
+	(./code/gradlew -p code clean)
+
+format:
+	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) spotlessApply)
```

Comment:
Let's also apply spotless to the `TEST-APP-FOLDER-NAME`

Reply:
Good call